modules/update-preparing-ack.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-prepare.adoc
+
+:_content-type: PROCEDURE
+[id="update-preparing-ack_{context}"]
+= Providing the administrator acknowledgment
+
+After you have evaluated your cluster for any removed APIs and have migrated any removed APIs, you can acknowledge that your cluster is ready to upgrade from {product-title} 4.11 to 4.12.
+
+[WARNING]
+====
+Be aware that all responsibility falls on the administrator to ensure that all uses of removed APIs have been resolved and migrated as necessary before providing this administrator acknowledgment. {product-title} can assist with the evaluation, but cannot identify all possible uses of removed APIs, especially idle workloads or external tools.
+====
+
+.Prerequisites
+
+* You must have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+* Run the following command to acknowledge that you have completed the evaluation and your cluster is ready to upgrade to {product-title} 4.12:
```

Comment:
nit: there are potentially other issues with 4.11-to-4.12 exposure besides the Kube API removals.  Maybe we should soften this from "your cluster is ready to upgrade to {product-title} 4.12" to "your cluster is ready for the {product-title} 4.12 Kubernetes API removals" or some such?

Reply:
Thanks @wking, I updated to "your cluster is ready for the Kubernetes API removals in {product-title} 4.12:"